addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -293,7 +293,7 @@ var FileWidget = SearchableMediaWidget.extend({
         // if the document is not yet created, do not see the documents of other users
         if (!this.options.res_id) {
             attachedDocumentDomain.unshift('&');
-            attachedDocumentDomain.push(['create_uid', '=', this.options.user_id]);
```

Comment:
Was this option used at some point? Should not it be set correctly instead of using the session here directly? Not sure.

Reply:
> Was this option used at some point? Should not it be set correctly instead of using the session here directly? Not sure.

we couldn't find the definition or any such use case where this option used. we need to pass 'user_id' on everyplace in options when we call the open media dialog in web_editor. 
Moreover, session.uid is also used in this file below the line. 'relatedDomain.push(['create_uid', '=', session.uid])'

Reply:
You are right :+1: Damn editor... I'll merge this monday